app/screens/custom_status/custom_status_modal.tsx

```diff
@@ -264,6 +358,41 @@ class CustomStatusModal extends NavigationComponent<Props, State> {
                 )}
             </TouchableOpacity>
         );
+        const renderClearAfterTime = duration !== null && duration !== undefined && duration !== CustomStatusDuration.DATE_AND_TIME ? (
```

Comment:
`duration !== null && duration !== undefined` can be simplified to one of those options:

1. `Boolean(duration)`
2. `!!duration`
3. `duration`

If there is a possibility for duration to be `0` you would have to add a check for it being exactly `0` as well, since that would be falsey